spec/support/controller_spec_compatibility.rb

```diff
@@ -0,0 +1,18 @@
+module ControllerSpecCompatibility
+
+  # Provide keyword argument HTTP get method for controller specs
+  # when testing against Rails versions before 5.0
+  # if Rails::VERSION::MAJOR < 5
+  #   def get(path, params: nil, headers: nil)
+  #     super(path, params: params, headers: headers)
+  #   end
+  # end
+
+  def permitted_params(params)
+    ActionController::Parameters.new(params).permit!
+  end
+end
+
+RSpec.configure do |config|
```

Comment:
> Is this just a temporary testing shim?

Yep! Needs work, its hopefully going to be a way to stop Rails 5.0 deprecation notices for `get`, `post` etc. calls in controller specs that don't use keyword args.